Gusto/gusto.github.io
# Fix payrolls totals documentation to be consistent

## Description

The previous documented totals were likely made-up numbers and Brian Vu requested that we use total output so that everything adds up properly.

## Changes

- Fix payrolls totals documentation to be consistent

## Testing

* Execute `bundle exec jekyll serve`
* Visit http://localhost:4000/v1/payrolls
* Observe changed totals
